python/lsst/ts/observatory/control/maintel/mtcs.py

```diff
@@ -632,11 +637,490 @@ async def get_bore_sight_angle(self):
+
+    async def reset_m1m3_forces(self):
+        """Reset M1M3 forces."""
+
+        forces = np.zeros_like(
+            self.rem.mtm1m3.cmd_applyAberrationForces.DataType().zForces
+        )
+        await self.rem.mtm1m3.cmd_applyAberrationForces.set_start(
+            zForces=forces, timeout=self.fast_timeout
+        )
+        await self.rem.mtm1m3.cmd_applyActiveOpticForces.set_start(
+            zForces=forces, timeout=self.fast_timeout
+        )
+
+    async def enable_m2_balance_system(self):
+        """Enable m2 balance system."""
+
+        m2_force_balance_system_status = (
+            await self.rem.mtm2.evt_forceBalanceSystemStatus.aget(
+                timeout=self.fast_timeout
+            )
+        )
+
+        self.rem.mtm2.evt_forceBalanceSystemStatus.flush()
+
+        if not m2_force_balance_system_status.status:
+            self.log.debug("Enabling M2 force balance system.")
+            await self.rem.mtm2.cmd_switchForceBalanceSystem.set_start(
+                status=True, timeout=self.long_timeout
+            )
+            await self.rem.mtm2.evt_forceBalanceSystemStatus.next(
+                flush=False, timeout=self.long_timeout
+            )
+        else:
+            self.log.info("M2 force balance system already enabled. Nothing to do.")
+
+    async def reset_m2_forces(self):
+        """Reset M2 forces."""
+        await self.rem.mtm2.cmd_resetForceOffsets.start(timeout=self.long_timeout)
+
+    async def enable_compensation_mode(self, component):
+        """Enable compensation mode for one of the hexapods.
+
+        Parameters
+        ----------
+        component : `str`
+            Name of the component. Must be in `compensation_mode_components`.
+
+        See Also
+        --------
+        disable_compensation_mode: Disable compensation mode.
+        compensation_mode_components: Set of components with compensation mode.
+        """
+
+        await self._handle_set_compensation_mode(component, enable=True)
+
+    async def disable_compensation_mode(self, component):
+        """Disable compensation mode for one of the hexapods.
+
+        Parameters
+        ----------
+        component : `str`
+            Name of the component
+
+        See Also
+        --------
+        enable_compensation_mode: Enable compensation mode.
+        compensation_mode_components: Set of components with compensation mode.
+        """
+
+        await self._handle_set_compensation_mode(component, enable=False)
+
+    async def _handle_set_compensation_mode(self, component, enable):
+        """Handle setting compensation mode.
+
+        Parameters
+        ----------
+        component : `str`
+            Name of the component
+        enable : `bool`
+            Whether to enable or disable compensation mode.
+
+        Raises
+        ------
+        AssertionError
+            If `component` does not support compensation mode is not a valid
+            component.
```

Comment:
I don't understand this sentence. Is there perhaps an `or` missing between `mode` and `is`?